### PR TITLE
Optimize integration test.

### PR DESCRIPTION
**Pull request by 3pointer**
### What problem does this PR solve? 
Optimize integration test.

### What is changed and how it works?
br_full_ddl: adjust record count from 10000 -> 1000
### Check List 

Tests 

 - Unit test
 - Integration test

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Release note

 - No Release note.

### CODE REVIEW

**Comment by 3pointer** [comment]
unstable test: https://ci.pingcap.net/blue/organizations/jenkins/br_ghpr_unit_and_integration_test/detail/br_ghpr_unit_and_integration_test/5264/pipeline

**Comment by 3pointer** [comment]
unstable test:  lightning_parquet_checkpoint https://ci.pingcap.net/blue/organizations/jenkins/br_ghpr_unit_and_integration_test/detail/br_ghpr_unit_and_integration_test/5281/pipeline/

**Comment by glorv** [comment]
> unstable test: lightning_parquet_checkpoint https://ci.pingcap.net/blue/organizations/jenkins/br_ghpr_unit_and_integration_test/detail/br_ghpr_unit_and_integration_test/5281/pipeline/

fixed in #1286

**Comment by fengou1** [comment]
LGMT

**Comment by fengou1** [comment]
LGTM

File: tests/br_tikv_outage2/run.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#! /bin/bash
+
+set -eux
+
+. run_services
+
+wait_file_exist() {
```
Comment (glorv) [inline tests/br_tikv_outage2/run.sh:7]: shall we reuse the functions in `br_tikv_outage/run.sh`

**Review by glorv — COMMENTED** [review]
rest LGTM

File: tests/br_tikv_outage/run.sh [hunk 2]
```diff
@@ -60,28 +14,24 @@ hint_get_backup_client=$TEST_DIR/hint_get_backup_client
 
 # NOTE : cases `outage-at-finegrained shutdown scale-out` should be first to avoid issue
 #       https://github.com/pingcap/br/issues/1050
-cases=${cases:-'outage-at-finegrained shutdown scale-out outage outage-after-request'}
+cases=${cases:-'shutdown scale-out'}
```
Comment (kennytm) [inline tests/br_tikv_outage/run.sh:17]: the NOTE is outdated.

**Review by kennytm — COMMENTED** [review]
rest LGTM

File: tests/br_tikv_outage2/run.sh [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+load
+
+hint_finegrained=$TEST_DIR/hint_finegrained
+hint_backup_start=$TEST_DIR/hint_backup_start
+hint_get_backup_client=$TEST_DIR/hint_get_backup_client
+
+# NOTE : cases `outage-at-finegrained shutdown scale-out` should be first to avoid issue
+#       https://github.com/pingcap/br/issues/1050
+cases=${cases:-'outage-at-finegrained outage outage-after-request'}
```
Comment (kennytm) [inline tests/br_tikv_outage2/run.sh:17]: (ditto)

**Review by glorv — APPROVED** [review]

**Review by zwj-coder — APPROVED** [review]

**Pull request merged**